lib/ansible/modules/cloud/amazon/ec2_group.py

```diff
@@ -534,9 +569,18 @@ def main():
     else:
         module.exit_json(changed=changed, group_id=None)
 
-# import module snippets
-from ansible.module_utils.basic import *
-from ansible.module_utils.ec2 import *
+
+import re
```

Comment:
imports should all be just after the documentation sections (effectively at the top of the actual non-doc code)